frontend/src/component/admin/groups/Group/Group.tsx

```diff
@@ -118,13 +118,7 @@ export const Group: VFC = () => {
             {
                 Header: 'Last login',
                 accessor: (row: IGroupUser) => row.seenAt || '',
-                Cell: ({ row: { original: user } }: any) => (
-                    <TimeAgoCell
-                        value={user.seenAt}
-                        emptyText='Never'
-                        title={(date) => `Last login: ${date}`}
```

Comment:
You mentioned it being redundant. Is it in the sense that the cell content says the same? This might be an accessibility thing. Not sure if removing it is the best, unless TimeAgoCell also has the title property

Reply:
I understand, but if you use a screen reader, it may work better if you have the full "Last login: 07/01/2025".

Reply:
If that's the case, I vote we go for consistency the other way around: https://github.com/Unleash/unleash/pull/10475/commits/d032c2e07f5f3c353fcd4afe28e947e14b8290a1